app/commands/user/authenticate_from_omniauth.rb

```diff
@@ -20,7 +20,9 @@ def find_by_uid
       end
 
       user.update_column(:github_username, auth.info.nickname)
-      user
+      user.tap do
+        AwardPullRequestReputationJob.perform_later(user)
+      end
```

Comment:
This is going to happen every time someone logs in regardless of whether their username has changed or not.

I think we should wrap all of 22-25 in `if github_username != auth.info.nickname`